src/main/java/com/nike/cerberus/domain/cloudformation/CmsParameters.java

```diff
@@ -38,12 +38,12 @@
     @JsonUnwrapped
     private LaunchConfigParametersDelegate launchConfigParameters = new LaunchConfigParametersDelegate();
 
-    public String getBaseStackName() {
-        return baseStackName;
+    public String getCmsIamRoleName() {
```

Comment:
Why is this changed?

Reply:
You can not do x-region stack reference imports, so we have to explicitly provide via a parameter the thing (the CMS IAM role) we used to import in CloudFormation from the base stack.

Reply:
IAM roles are global but created in the primary region and we want to use said iam role in the secondary region CMS stack.